### ISSUE TITLE
 I cannot reproduce the results of the paper

### ISSUE THREAD

**Issue by zc-zhao** [valeoai/rangevit#6]
When i initialize RangeViT’s backbone with  ViTs pretrained on supervised ImageNet21k classification and fine-tuned on supervised image segmentation on Cityscapes with Segmenter (entry Cityscapes),I cannot reproduce the results of the paper regarding the nuScenes dataset .
My training configuration is as follows:
`two GeForce RTX 4090 ,the batch-size of each GPU is sixteen .Other configurations follow your config file.`
The environment is as follows:
`pytorch==2.0.0,torchvision==1.15.1,cuda==11.7`
As a result, the miou only reached 74.27.
Could you tell me why this situation happened?

**Comment by YangParky** [comment]
Hi, have you tested his published model?

**Comment by zc-zhao** [comment]
> Hi, have you tested his published model?

Yes,i have tested his published model and obtained a structure consistent with the paper.But the results of my own training did not meet the requirements of the paper.

**Comment by manthan99** [comment]
Hello

I am also having similar issues where I am unable to reproduce the results. I am using 4 RTX 2080s with a batch size of 8 and am able to get 74.75 instead of 75.2, as mentioned for the CityScapes pre-trained model.
